src/pages/register.tsx

```diff
@@ -1,91 +1,107 @@
 import { useContext, useState } from "react";
-
 import { useRouter } from "next/router";
-
 import { Dayjs } from "dayjs";
 
 import { userContext } from "supabase/user_context";
 import { useSupabaseClient } from "@supabase/auth-helpers-react";
 import { Database } from "supabase/db_types";
-
-import { Box, Stack, Button, SelectChangeEvent, Grid } from "@mui/material";
+import { Box, Stack, Button, SelectChangeEvent, Grid, Card, Typography } from "@mui/material";
+import { grey } from "@mui/material/colors";
 
 import Loading from "@/components/public/Loading";
+import Background from "@/components/public/Background";
 import Logo from "@/components/public/Logo";
 import CommonTextField from "@/components/public/CommonTextField";
 import PasswordTextFeild from "@/components/public/PasswordTextField";
-import CommonDropdown from "@/components/public/CommonDropdown";
+import CommonDropdown from "@/components/public/GenderDropdown";
 import CommonDatePicker from "@/components/public/CommonDatePicker";
-
 import { validateEmail, validateTextField } from "@/utilities/validation";
+
 import { validation } from "@/types/Validation";
-import { Gender } from "enum/gender";
-import { CHAR_LIMIT } from "enum/inputLimit";
-import { PagePaths } from "enum/pages";
+import { GENDER } from "enum/GENDER";
+import { CHAR_LIMIT } from "enum/INPUT_LIMIT";
+import { PAGE_PATHS } from "enum/PAGES";
+import { ICONS } from "enum/ICONS";
+
 import { CreateUser } from "@/services/User";
+import NormalButton from "@/components/public/CommonButton";
+
+type RegisterInput = {
+  displayName: string;
+  email: string;
+  password: string;
+  confirmPassword: string;
+  gender: string;
+  birthDate: Dayjs | null;
+};
+
+type RegisterSubmit = {
+  displayName: boolean;
+  email: boolean;
+  password: boolean;
+  confirmPassword: boolean;
+  gender: boolean;
+  birthDate: boolean;
+};
 
-// style
-const register_layout = {
-  width: "35vw",
-  minWidth: "350px",
+const RegisterStyle = {
+  Card: {
+    width: "50vw",
+    minWidth: "300px",
+    minHeight: "200px",
+
+    paddingTop: "2vh",
+    paddingBottom: "1vh",
+
+    backgroundColor: grey[300],
+  },
+  TextField: {
+    width: "35vw",
+    minWidth: "350px",
+  },
 };
```

Comment:
cannot scroll
<img width="1552" alt="image" src="https://user-images.githubusercontent.com/62459518/226545971-4b9b7db9-dafa-45a0-a5c8-4fd49f5b9943.png">

Reply:
Solved

Reply:
The upper part of the page is missing. (Pic below is the top of the page)
<img width="1552" alt="image" src="https://user-images.githubusercontent.com/62459518/226829638-2a292d46-7a7d-4631-8344-6001abeaf66a.png">

Reply:
Solved this problem by using margin instead of center attribute; however, this causes the page look like this when zoom out. Does this acceptable?
![image](https://user-images.githubusercontent.com/69895436/226849811-471d226d-6c0f-4dcf-ac5a-8479d9522df7.png)

Reply:
accept ka